starknet-replay/src/runner/mod.rs

```diff
@@ -30,6 +29,8 @@ pub mod report;
 ///   transactions.
 /// - `storage`: The object to query the starknet blockchain using the RPC
 ///   protocol.
+/// - `serial_replay`: Set to true to do serial blocks replay instead of
+///   parallel.
```

Comment:
You provide details in downstream functions (`replay_blocks_*`) that should probably be provided here about the impacts of each kind of choice in this case.